engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/ProcessDefinitionExtractor.scala

```diff
@@ -123,6 +123,18 @@ object ProcessDefinitionExtractor {
       ids.toList
     }
 
+    @Override
+    def categories: List[String] = {
```

Comment:
maybe let's remove it now as it's not used?

Reply:
Removed